monai/transforms/utils_create_transform_ims.py

```diff
@@ -280,22 +279,24 @@ def get_stacked_before_after(before, after, is_label=False):
     return [get_stacked_2d_ims(d, is_label) for d in (before, after)]
 
 
-def save_image(images, labels, filename, transform_name, transform_args, im_sizes, colorbar=False):
+def save_image(images, labels, filename, transform_name, transform_args, shapes, colorbar=False):
     """Save image to file, ensuring there's no whitespace around the edge."""
     plt.rcParams.update({"font.family": "monospace"})
     plt.style.use("dark_background")
-    fig, axes = plt.subplots(len(images), len(images[0]))
-    for row in range(len(images)):
+    nrow = len(images)  # before and after (should always be 2)
+    ncol = len(images[0])  # num orthogonal views (either 1 or 3)
+    fig, axes = plt.subplots(nrow, ncol)
+    for row in range(nrow):
         vmin = min(i.min() for i in images[row])
         vmax = max(i.max() for i in images[row])
-        for col in range(len(images[0])):
-            ax = axes[row][col]
+        for col in range(ncol):
+            ax = axes[row][col] if ncol > 1 else axes[row]
             imshow = ax.imshow(images[row][col], cmap="gray", vmin=vmin, vmax=vmax)
```

Comment:
might have some minor issues - the padding transform figure doesn't show the actual size of the images, looks like both before and after are scaled to the same panel size. - for the label vis, maybe the option of no antialiasing should be used to keep the values discrete?

Reply:
I struggled to get the sizes of the figures to reflect the sizes of the image. Instead I displayed the image sizes in the y_label, see [here](https://docs.monai.io/en/latest/transforms.html#borderpadd). I messed with the `gridspec_kw` argument to `imshow`, but didn't get perfect results. Feel free to have a look though. 

Good idea for the anti-aliasing.